app/controllers/race_entries_controller.rb

```diff
@@ -1,45 +1,21 @@
 class RaceEntriesController < ApplicationController
-  def index
-    sort = params[:sort] || 'racers.last_name'
-    @race_entries = RaceEntry.includes(:racer, :race_edition).order(sort).map { |re| RaceEntryPresenter.new(re) }
-  end
-  
-  def show
```

Comment:
This was identical to the `#edit` action. Because both `#edit` and `#show` took the user to a form from which the record could be edited, I thought it was appropriate to retain and use the `#edit` action and delete the `#show` action.